compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -200,15 +195,6 @@ def __init__(
                     "look like they might point to different environments.  "
                     "Double check that they are the correct URLs."
                 )
-
-            # Start up an asynchronous polling loop in the background
-            self.ws_polling_task = WebSocketPollingTask(
-                self,
-                self.loop,
-                init_task_group_id=self.session_task_group_id,
-                results_ws_uri=self.results_ws_uri,
-            )
-        else:
```

Comment:
Excellent!  I was going to make a new PR for this, but I could be convinced to sneak it into this one.  However, this is only a partial set of this work, so please either complete it or make this into a separate PR.  Outstanding items in my mind for this change:

- Certainly a changelog entry
- Removal of `urls_might_mismatch`
- Removal of `warn_about_url_mismatch`
- Removal of `Client.results_ws_uri`
- Removal of `Client.asynchronous`
- Removal of `Client.loop`
- Removal of the deprecation notices
- Constructor docstring update (which similarly affects the generated documentation)
- Removal of `AtomicController`
- Accordingly updating all of the tests

Reply:
This was only removed due to the removal of `Client.session_task_group_id`. Your point is well-taken, so I've reverted the removal of `ws_polling_task` and `session_task_group_id`; this is the only location where the latter is referenced.